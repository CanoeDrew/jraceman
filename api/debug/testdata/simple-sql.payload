{
"q": "select name,bar from test where foo='x' order by id"
}
