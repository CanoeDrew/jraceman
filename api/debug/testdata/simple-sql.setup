CREATE table test(id int, name string not null, foo string, bar bool);

INSERT into test(id, name, foo, bar) values
(1, 'a', 'x', false),
(2, 'b', 'x', true),
(3, 'c', null, false);
