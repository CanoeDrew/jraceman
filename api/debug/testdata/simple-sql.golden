{
  "Columns": [
    {
      "Name": "name",
      "Type": "string"
    },
    {
      "Name": "bar",
      "Type": "bool"
    }
  ],
  "Rows": [
    [
      "a",
      0
    ],
    [
      "b",
      1
    ]
  ]
}